.pre-commit-config.yaml

```diff
@@ -34,7 +34,7 @@ repos:
         additional_dependencies:
           - toml
   - repo: https://github.com/myint/docformatter
-    rev: 'v1.5.0'
+    rev: 'v1.5.1'
```

Comment:
Docformatter 1.5.1 has a bug that affects formatting of newlines after class definitions, see https://github.com/PyCQA/docformatter/issues/139

I suggest we stick to 1.5.0 until a new version of docformatter is available.

Reply:
👍